app/model.py

```diff
@@ -139,15 +143,20 @@ def list(cls):
     @classmethod
     def list_active(cls):
         """Returns a list of the active (non-deactivated) repository names."""
-        return [name for name in Repo.list()
-                if not config.get_for_repo(name, 'deactivated')]
+        staging = [repo.key().name() for repo in Repo.all().filter(
```

Comment:
You can't add generators like that (at least not in Python 2? dunno about Python 3). Since this function isn't used by any end-user-facing pages, I think it's ok to sacrifice a little efficiency in favor of simplicity.